### DISCUSSION TITLE
[Feature] Better Keyboard Shortcuts 2.0

### DISCUSSION THREAD

**Discussion by Coldblackice** [File-New-Project/EarTrumpet#1289] [Ideas]
The ability to keyboard shortcut between devices/app volumes is great, but if the user has a number of devices and/or apps, it becomes slower than using the mouse. A couple ideas:

- **Volume-change increment modifier**: 
Currently, arrow-keys change the volume in 1% increments, however, adding a keyboard-modifier option, e.g. holding CTRL|SHIFT|ALT would (temporarily) increase the increment to 10-20% jumps
- **Quick-focusing apps in panel**: 
Similar to quick-selection in Windows explorer/context menus, EarTrumpet's panel list of apps could be quick-selected by typing the first letter(s) of the app name, e.g. _Chrome_-_Firefox_-_Opera_-_Vivaldi_ --> user presses "F" --> selector focuses _Firefox_'s volume
If >1 apps have the same first letter, it follows Windows/Explorer functionality by jumping to the first app (unless additional letters are typed), a pause then re-typing the same letter moves the selector to the next app
- **Mute/unmute volume of currently-focused app**
- **Mute all other apps except currently-focused app**
- **Option for giving a macro _Action_ a keyboard shortcut**

**Comment by halaczkiewiczk** [comment]
There are more features on the [dev release](https://install.eartrumpet.app/dev/EarTrumpet.Package.appinstaller)

**Comment by philippeop** [comment]
Anyone looking for a specific hotkey, the dev version has a 'Actions' menu that lets you create custom hotkeys.
Just press the + button beneath the 'Find a setting' bar.

**Comment by EvilGeniusSkis** [comment]
In the actions menu on the dev release, can we get actions to; increase/decrease volume of the foreground app, change playback device of the foreground app to the next/prev in the list, change playback to "any (bluetooth/Hdmi/USB etc.)", "ALL" as an option for devices (Eg: mute the foreground app on all devices)?
